game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -42,17 +45,16 @@ public GameSelectorModel() {
   }
 
   public void resetGameDataToNull() {
-    setGameData(null);
-    fileName = "";
+    load(null, null);
   }
 
-  public void load(final GameData data, final String fileName) {
+  public void load(final @Nullable GameData data, final @Nullable String fileName) {
     setGameData(data);
     this.fileName = fileName;
   }
 
   public void load(final GameChooserEntry entry) {
-    fileName = entry.getLocation();
```

Comment:
No it's not.
This would be the part where the filename, which ultimately gets applied to the label is set.
So setting it to null ensures that the savegame is cleared when a new map is loaded